labels.json

```diff
@@ -1,135 +1,316 @@
+    "name": "resolution: invalid",
+    "description": "An Issue that is determined to not be related to this project",
+    "color": "e6e6e6",
     "aliases": [
-      "feedback",
-      "needs-feedback"
+      "resolution: invalid",
+      "invalid"
     ]
   },
   {
-    "name": "needs-help",
-    "color": "d4c5f9",
+    "name": "resolution: won't fix",
+    "description": "An Issue reporting behaviour that is not intended to be changed in the future",
+    "color": "ffffff",
     "aliases": [
-      "help",
-      "help wanted",
-      "needs-help"
+      "resolution: won't fix",
+      "wont-fix",
+      "wontfix",
+      "wont-fix",
+      "won't-fix"
     ]
   },
   {
-    "name": "needs-rebase",
+    "name": "resolution: works as expected",
+    "description": "An Issue reporting a bug that is actually normal behaviour",
     "color": "207de5",
     "aliases": [
-      "needs-rebase",
-      "rebase"
+      "resolution: works as expected",
+      "not-a-bug"
+    ]
+  },
+  {
+    "name": "keep open",
+    "description": "An Issue or Pull Request that should be kept open",
+    "color": "fbca04",
+    "aliases": [
+      "keep open",
+      "leave-open"
     ]
   },
   {
-    "name": "needs-squash",
+    "name": "for new contributors",
+    "description": "An Issue raising a need for changes, that should be relatively easy to do",
+    "color": "02d7e1",
+    "aliases": [
+      "for new contributors",
+      "beginner",
+      "beginner-friendly",
+      "beginner friendly"
+    ]
+  },
+  {
+    "name": "status: confirmed",
+    "description": "An Issue reporting a bug that is confirmed to affect other users",
     "color": "c2e0c6",
     "aliases": [
-      "needs-squash",
-      "squash"
+      "status: confirmed",
+      "confirmed"
     ]
   },
   {
-    "name": "needs-tests",
-    "color": "f7c6c7",
+    "name": "status: unconfirmed",
+    "description": "An Issue reporting a potential bug that is yet to be confirmed by other users",
+    "color": "fef2c0",
+    "aliases": [
+      "status: unconfirmed",
+      "unconfirmed"
+    ]
+  },
+  {
+    "name": "security",
+    "description": "Any Issue or Pull Request that directly relates to security",
+    "color": "ff1d21",
+    "aliases": [
+      "security"
+    ]
+  },
+  {
+    "name": "type: chore",
+    "description": "A Pull Request that is raised as part of routine module maintenance",
+    "color": "e7ed39",
+    "aliases": [
+      "type: chore",
+      "chore",
+      "modulesync"
+    ]
+  },
+  {
+    "name": "type: feature",
+    "description": "A Pull Request raised that adds completely new functionality",
+    "color": "c7def8",
+    "aliases": [
+      "type: feature",
+      "feature"
+    ]
+  },
+  {
+    "name": "type: enhancement",
+    "description": "A Pull Request raised that enhances existing functionality",
+    "color": "84b6eb",
+    "aliases": [
+      "type: enhancement",
+      "enhancement"
+    ]
+  },
+  {
+    "name": "type: refactor",
```

Comment:
Not always. The idea is that the changes require more scrutiny than usual as many moving parts are affected, increasing the risk of a bad change being introduced.